Dockerfile

```diff
@@ -50,4 +45,9 @@ RUN CHROME_MAJOR_VERSION=$(google-chrome --version | sed -E "s/.* ([0-9]+)(\.[0-
   && rm /tmp/chromedriver_linux64.zip \
   && chmod 755 /usr/bin/chromedriver
 
-ENTRYPOINT ["poetry", "run", "pytest"]
\ No newline at end of file
+ENTRYPOINT ["poetry", "run", "pytest"]
+
+FROM base as production
+
+ENTRYPOINT ["gunicorn"]
+CMD ["todo_app.app:create_app()"]
```

Comment:
Minor, but note that your switch here has changed the default behaviour slightly to running on port 8000 (Gunicorn's default) rather than 5000 (as you used to specify), which means your Readme instructions for running the prod container are out of date